.travis.yml

```diff
@@ -17,7 +18,24 @@ before_script:
   - composer selfupdate --quiet
   - composer install --prefer-dist --no-interaction --no-progress
   - ./vendor/bin/phpunit
-  - composer update --prefer-dist --no-interaction --no-progress
 
 script:
-  - ./vendor/bin/phpunit
+  - make test
+
+after_script:
```

Comment:
Sorry, I have commented without seeing your changes...
Is the `[ "${TRAVIS_PULL_REQUEST}" = "false" ]` condition really necessary? From what I have read in the Travis documentation the deploy only occurs for the master branch by default. It should not run on Pull Requests?